bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -169,6 +169,15 @@ def test_update_workflow_name(self):
         updated_wf = self.gi.workflows.update_workflow(wf['id'], name=new_name)
         self.assertEqual(updated_wf['name'], new_name)
 
+    @test_util.skip_unless_galaxy('release_18.05')
+    def test_update_workflow_published(self):
+        path = test_util.get_abspath(os.path.join('data', 'paste_columns.ga'))
+        wf = self.gi.workflows.import_workflow_from_local_path(path)
```

Comment:
```suggestion
        self.assertFalse(updated_wf['published'])
        wf = self.gi.workflows.import_workflow_from_local_path(path)
```

Reply:
Just for completeness - the workflow should be unpublished by default.

Thanks for the PR @rikeshi.